messages/en.json

```diff
@@ -548,7 +548,8 @@
     "spaceBetween": "Space between",
     "output": "Output",
     "appendFileNames": "Append file names with tool names.",
-    "example": "Example:"
+    "example": "Example:",
+    "appendCustomNames": "Append a custom name to the file."
```

Comment:
Let's change it to:
"Append file names with custom text."

Then, translate to other languages again.